libraries/Microsoft.Bot.Builder.Dialogs/DialogStatus.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.Dialogs
+{
+    public enum DialogStatus
```

Comment:
Should this maybe be called DialogTurnStatus?

Reply:
I have no issue with that.